staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -35,12 +35,17 @@ import (
 
 var (
 	explainLong = templates.LongDesc(i18n.T(`
-		Describe fields and structure of various resources.
+		Describe (sub-)fields and structure of various resources.
```

Comment:
I question how useful it is to denote subfields here. All the subfields are just simply fields of other fields, and it is demonstrated in the example text that there can be any number of fields that are children of resource fields.